aminator/plugins/provisioner/apt.py

```diff
@@ -89,19 +81,37 @@ def _store_package_metadata(self):
 
     @staticmethod
     def dpkg_install(package):
-        return monitor_command(['dpkg', '-i', package])
+        dpkg_result = monitor_command(['dpkg', '-i', package])
+        if not dpkg_result.success:
+            log.debug('failure:{0.command} :{0.std_err}'.format(dpkg_ret.result))
+        return dpkg_result
 
     @classmethod
-    def apt_get_localinstall(cls, package):
+    def _fix_localinstall_deps(cls, package):
```

Comment:
localinstall for apt vs aptitude only varied in how it handled fixing dependencies. both start with dpkg -i, so I teased the dep foo out into separate methods and kept common general install logic in _localinstall()